server/dining/balance.py

```diff
@@ -97,3 +98,28 @@ def get_dining_balance():
         }})
     else:
         return jsonify({'balance': None})
+
```

Comment:
We conform to Flake styling throughout the server. Need two spaces between functions, so add one here. For the rest of the flake errors, check out https://travis-ci.org/pennlabs/labs-api-server/jobs/585255472.